xapian-core/weight/tfidfweight.cc

```diff
@@ -210,6 +213,14 @@ TfIdfWeight::get_wdfn(Xapian::termcount wdf, Xapian::termcount doclen,
 	    double den = 1 + log(wdf_avg);
 	    return num / den;
 	}
+	case 'a': {
+	    if (wdf == 0) return 0;
+	    return (0.2 + 0.8 * log(1.0 + wdf));
```

Comment:
This doesn't seem to match the definition of "a" for the first normalisation at http://people.csail.mit.edu/jrennie/ecoc-svm/smart.html : 

> "**a**ug-norm" : new-tf = 0.5 + 0.5 * (tf / max-tf)
>              augmented normalized tf.  0.5 < new-tf <= 1.0

Reply:
'A' is for Augmented log (I have changed from 'a' to 'A') .